meta-balena-common/recipes-core/images/balena-image.bb

```diff
@@ -58,10 +58,23 @@ generate_hostos_version () {
     echo "${HOSTOS_VERSION}" > ${DEPLOY_DIR_IMAGE}/VERSION_HOSTOS
 }
 
+symlink_image_signature () {
+    # This is probably not the correct way to do it, but it works.
+    # We sign BALENA_RAW_IMG, which ends up in IMGDEPLOYDIR
+    # and has a timestamp in the file name. We need to get rid
+    # of the timestamp for the final deploy, so that the file
+    # ends up in a predictable location.
+
+    ln -sf "${BALENA_RAW_IMG}.sig" "${DEPLOY_DIR_IMAGE}/balena-image-${MACHINE}.balenaos-img.sig"
```

Comment:
I am still not 100% confident about this part, it might be poisoning the cache with an old signature, but I didn't find a better way to do it.

Reply:
The current implementation should work - in case we find out it doesn't another way to approach this would have been to have a `balenaos-img.sig` image type that both signed and symlinked the final images.